services/simcore/docker-compose.yml.j2

```diff
@@ -227,7 +227,7 @@ services:
         - traefik.http.routers.${SWARM_STACK_NAME}_webserver.middlewares=${SWARM_STACK_NAME}_gzip@swarm, ${SWARM_STACK_NAME_NO_HYPHEN}_sslheader
         - traefik.http.routers.${SWARM_STACK_NAME}_webserver_swagger.service=${SWARM_STACK_NAME}_webserver
         - traefik.http.routers.${SWARM_STACK_NAME}_webserver_swagger.middlewares=${SWARM_STACK_NAME}_gzip@swarm, ${SWARM_STACK_NAME_NO_HYPHEN}_sslheader
-        - traefik.http.routers.${SWARM_STACK_NAME}_webserver_swagger.rule=PathPrefix(`/dev/`)
+        - traefik.http.routers.${SWARM_STACK_NAME}_webserver_swagger.rule=PathPrefix(`/dev/`) || PathPrefix(`/doc/`)
```

Comment:
thx. 
Q: is `/doc/` the same as `/doc`? If not, then, as I mentioned in the [issue](https://github.com/ITISFoundation/osparc-ops-environments/issues/1033) i would like to have the latter

Reply:
Q: Is there an easy way to test this before you deploy it in master?

Reply:
allowed `/doc/` and `/doc` and tested it locally